test-cases/longevity/longevity-large-partition-4days.yaml

```diff
@@ -24,3 +24,5 @@ space_node_threshold: 644245094
 validate_partitions: true
 table_name: "scylla_bench.test"
 primary_key_column: "pk"
+
+run_full_partition_scan: '{"ks_cf": "scylla_bench.test", "interval": 2, "pk_name":"pk", "rows_count": 10000000}' # 'ks.cf, interval(min), partition-key name, number-of-rows-per-partition'
```

Comment:
now need to understand the impact of running it in a long test like this, and for so many days, and see if it won't disturb the whole load.

Reply:
will start running the heavier longevities soon.

Reply:
started 4d longevity in: https://jenkins.scylladb.com/job/scylla-staging/job/Longevity_yaron/job/longevity-large-partition-asymmetric-cluster-3h_reversed_queries/40/

Reply:
Updated results are in:
https://docs.google.com/spreadsheets/d/1L3n-HwSi97_h0tOmDlkB1r69wz9NQYSYDDUXjqnf8YQ/edit?usp=sharing